zadaniee3.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+
+public class NumberAnalysis
+{
+    public static void Main(string[] args)
+    {
+        Console.WriteLine("Введите четырехзначное число:");
+        int number = Convert.ToInt32(Console.ReadLine());
+       
+            int thousands = number / 1000;
```

Comment:
Пропущен оператор if. В таком виде программа не скомпилируется
```suggestion
        int number = Convert.ToInt32(Console.ReadLine());
       
       if (number <1000 || number >=10000)
       {
            int thousands = number / 1000;
```